City-of-Pori/pori-kada
# PORI-237 remove target group from carousel

NOTICE! Based on the branch here: https://github.com/City-of-Pori/pori-kada/pull/133 Merge it first before merging this.

To test:
1. drush fra -y  && drush cc all
2. Go to edit front page carousel liftups and add target groups to them.
3. Make sure the target group is not shown on the carousel.
